lib/query_builder/schema/comparison_filter.ex

```diff
@@ -38,10 +38,18 @@ defmodule EctoShorts.QueryBuilder.Schema.ComparisonFilter do
     where(query, [scm], is_nil(field(scm, ^filter_field)))
   end
 
+  defp build_subfield_filter(query, filter_field, :==, val) do
+    where(query, [scm], field(scm, ^filter_field) == ^val)
```

Comment:
This is unnecessary because you can just query directly `%{field: 1234}`

Reply:
100%.
It's an ~old~ bad habit of mine that inherited from working with languages like C# where you always have to provide implementations for both "operators pairs" `{==, !=}, {<,>}, et al..)`

I'll get rid of that one.